src/dom.js

```diff
@@ -32,9 +32,9 @@ const getPartials = (html = '', tag = 'svg') => {
   const result = [];
   html.replace(new RegExp(`<${tag}(?:\\s[^>]+)?>`, 'ig'), (match, offset, str) => {
     if (match.includes('/>')) {
-      result.push(str.substring(offset, offset + match.length));
+      result.push(str.slice(offset, offset + match.length));
```

Comment:
Double check this change @bezoerb

Reply:
LGTM. But why do you want to switch to slice in the first place?

Reply:
It's a new xo error https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-string-slice.md